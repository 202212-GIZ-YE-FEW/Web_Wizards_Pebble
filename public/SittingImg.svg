<svg width="371" height="404" viewBox="0 0 371 404" fill="none" xmlns="http://www.w3.org/2000/svg">
<path fill-rule="evenodd" clip-rule="evenodd" d="M155.231 44.2978C149.637 37.7317 145.851 29.6849 146.484 20.6682C148.307 -5.31137 184.471 0.403812 191.506 13.5161C198.541 26.6284 197.705 59.8887 188.613 62.2322C184.987 63.1668 177.263 60.8774 169.402 56.1391L174.337 91.0478H145.201L155.231 44.2978Z" fill="#B28B67"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M164.297 30.1879C163.344 29.5331 162.206 29.1353 160.983 29.1353C157.631 29.1353 154.913 32.1247 154.913 35.2052C154.913 38.41 156.427 40.8879 158.588 41.9391C157.433 44.1958 156.273 45.8335 154.913 46.131C149.667 45.6002 142.773 32.5594 142.773 21.8515C142.773 10.1312 152.099 0 169.481 0C185.104 0 191.027 6.4416 191.332 12.1397C184.638 13.9796 172.94 14.6881 156.127 14.5677V15.7816L170.695 16.9956L173.123 24.2795C169.26 23.9911 166.5 26.6406 164.297 30.1879Z" fill="#191847"/>
<path d="M133.061 403.04C184.686 403.04 226.537 363.092 226.537 313.813C226.537 264.534 184.686 224.586 133.061 224.586C81.4356 224.586 39.585 264.534 39.585 313.813C39.585 363.092 81.4356 403.04 133.061 403.04Z" fill="#C5CFD6"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M309.706 389.273C294.151 354.847 282.902 324.87 278.387 309.054C270.656 281.973 264.775 258.144 263.288 249.245C259.743 228.024 290.556 226.788 294.629 237.335C300.794 253.297 311.555 305.743 324.486 384.96L309.706 389.273ZM129.284 281.234C143.555 275.593 193.532 257.939 217.635 252.945C224.562 251.51 231.275 250.177 237.587 248.972C256.348 245.391 266.067 279.236 247.999 281.836C203 288.312 139.207 295 133.748 295.751C126.097 296.802 120.315 284.779 129.284 281.234Z" fill="#B28B67"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M249.101 239.659L171.908 200.363V191.809H181.088C255.769 210.838 293.931 223.034 295.574 228.395C295.592 228.453 295.608 228.511 295.623 228.568C295.66 228.618 295.697 228.667 295.733 228.717C310.534 249.072 319.397 348.444 322.307 355.885L293.946 360.115C283.098 319.887 248.389 291.862 249.311 243.68C249.062 242.267 248.999 240.927 249.101 239.659Z" fill="#1F28CF"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M178.322 262.997C171.259 261.583 164.754 260.321 159.189 259.345C123.239 253.039 112.756 233.795 118.954 191.809H185.191C196.011 194.917 252.242 218.498 281.416 231.04C300.726 239.341 294.639 269.266 283.704 277.053C283.645 277.531 283.448 277.855 283.097 278.001C224.275 302.38 177.741 289.588 162.196 293.676L155.006 270.994L178.322 262.997Z" fill="#2B44FF"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M134.163 277.63C130.309 277.998 127.573 278.577 125.955 279.366C123.987 280.326 121.276 282.054 117.821 284.551C118.934 286.833 128.939 307.347 147.836 346.091C153.903 345.112 156.307 342.797 155.049 339.145C153.79 335.494 152.774 332.446 152.002 330.002L150.803 295.179C150.78 294.509 150.218 293.984 149.548 294.007C149.537 294.008 149.525 294.008 149.514 294.009L144.157 294.346C141.113 291.882 139.105 289.653 138.132 287.659C137.342 286.038 136.905 283.482 136.821 279.988L136.821 279.988C136.789 278.648 135.676 277.587 134.336 277.619C134.278 277.621 134.22 277.624 134.163 277.63Z" fill="#E4E4E4"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M302.626 382.89C301.268 386.516 300.589 389.228 300.589 391.028C300.589 393.218 300.954 396.412 301.683 400.612C304.223 400.612 327.046 400.612 370.153 400.612C371.933 394.73 370.905 391.554 367.072 391.085C363.238 390.616 360.054 390.192 357.518 389.814L325.694 375.627C325.081 375.354 324.364 375.629 324.091 376.241C324.086 376.252 324.082 376.262 324.077 376.273L322.032 381.235C318.483 382.891 315.599 383.719 313.38 383.719C311.577 383.719 309.088 382.991 305.912 381.535L305.912 381.535C304.693 380.976 303.251 381.511 302.693 382.73C302.669 382.783 302.646 382.836 302.626 382.89Z" fill="#E4E4E4"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M244.613 167.688L283.336 196.124C293.839 198.958 302.614 202.484 309.66 206.702C311.325 208.218 313.337 211.29 306.433 210.759C299.53 210.228 292.245 209.98 291.293 212.335C290.34 214.69 294.517 217.505 292.592 220.682C291.308 222.8 285.244 217.49 274.399 204.75L236.052 188.849L244.613 167.688ZM69.3753 155.844L90.7548 159.715C65.9412 208.955 52.651 234.99 50.8842 237.82C46.9089 244.188 50.9219 255.662 52.4359 261.714C43.6391 263.925 46.8903 250.566 35.761 253.538C25.6028 256.25 17.0658 262.905 7.39926 254.3C6.21082 253.242 5.29577 249.886 9.25342 248.088C19.1134 243.607 33.6509 235.286 35.9763 232.113C39.1475 227.787 50.2805 202.364 69.3753 155.844Z" fill="#B28B67"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M253.634 170.779C258.616 170.421 264.036 171.246 269.846 173.204C272.705 174.167 274.242 177.266 273.279 180.125C272.719 181.786 271.439 183 269.904 183.538C271.016 184.349 272.138 185.156 273.27 185.958L263.5 205.901C222.392 185.404 183.754 169.951 166.147 136.112C160.238 124.755 162.238 86.1774 161.674 72.143L174.657 66.8184C192.756 104.229 217.249 140.12 253.634 170.779Z" fill="#69A1AC"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M116.065 202.623C154.231 206.815 184.271 206.815 206.186 202.623C208.901 202.103 208.782 190.399 207.523 187.202C187.915 137.423 175.016 100.982 175.016 66.4062C171.418 68.2856 165.122 68.5672 156.127 67.2512C131.952 102.882 120.839 143.765 116.065 202.623Z" fill="#191847"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M108.974 208.509C107.678 207.289 107.003 205.461 107.335 203.577L107.967 199.991C108.396 197.56 110.37 195.79 112.7 195.514C113.108 193.692 113.459 191.883 113.749 190.087C116.2 174.917 112.013 160.043 112.209 145.436C92.6886 162.091 75.1769 190.849 59.6743 231.712L36.7462 224.965C37.6211 221.715 38.5272 218.505 39.4648 215.334C36.8184 213.831 33.7261 206.94 35.9516 205.787C38.2501 204.596 40.685 203.758 43.2493 203.265C64.2721 139.945 98.6509 93.6366 146.386 64.3398H150.609L160.192 64.3398C156.127 118.095 212.273 235.455 192.777 335.685C156.743 343.732 105.3 318.695 55.2249 338.698C57.0882 298.334 94.4123 250.916 108.974 208.509Z" fill="#89C5CC"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M151.4 65.5547L145.136 84.8149L157.749 99.2534L146.35 110.6L173.927 157.817L140.204 110.694L151.4 99.2534L140.204 86.4938L151.4 65.5547Z" fill="black" fill-opacity="0.1"/>
<path fill-rule="evenodd" clip-rule="evenodd" d="M165.838 176.026C163.156 176.026 160.982 173.852 160.982 171.17C160.982 168.489 163.156 166.314 165.838 166.314C168.52 166.314 170.694 168.489 170.694 171.17C170.694 173.852 168.52 176.026 165.838 176.026ZM171.908 203.948C169.226 203.948 167.052 201.774 167.052 199.092C167.052 196.41 169.226 194.236 171.908 194.236C174.59 194.236 176.764 196.41 176.764 199.092C176.764 201.774 174.59 203.948 171.908 203.948ZM173.122 231.869C170.44 231.869 168.266 229.695 168.266 227.013C168.266 224.331 170.44 222.157 173.122 222.157C175.804 222.157 177.978 224.331 177.978 227.013C177.978 229.695 175.804 231.869 173.122 231.869Z" fill="black" fill-opacity="0.4"/>
</svg>
