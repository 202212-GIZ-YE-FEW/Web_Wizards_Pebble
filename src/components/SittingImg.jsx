import Image from "next/image";

import Sitting from "../../public/Sitting.png";

const SittingImg = () => {
    return (
        <div className='  mx-auto w-1/2 max-w-xl pl-16 lg:py-32 order-1'>
            <Image
                src={Sitting}
                alt='Sitting or a rock'
                className=' mx-auto  mt-16'
            />
        </div>
    );
};

export default SittingImg;
