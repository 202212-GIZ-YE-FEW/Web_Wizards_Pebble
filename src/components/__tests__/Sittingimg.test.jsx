import renderer from "react-test-renderer";

import SittingImg from "../SittingImg";

it("renders correctly", () => {
    const tree = renderer.create(<SittingImg />).toJSON();
    expect(tree).toMatchSnapshot();
});
