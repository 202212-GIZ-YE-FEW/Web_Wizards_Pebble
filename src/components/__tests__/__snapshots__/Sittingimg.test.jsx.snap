// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="  mx-auto w-1/2 max-w-xl pl-16 lg:py-32 order-1"
>
  <img
    alt="Sitting or a rock"
    className=" mx-auto  mt-16"
    data-nimg="1"
    decoding="async"
    height={40}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
    srcSet="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
    style={
      Object {
        "color": "transparent",
      }
    }
    width={40}
  />
</div>
`;
